package/base-files/files/lib/upgrade/stage2

```diff
@@ -127,6 +127,16 @@ killall -9 telnetd
 killall -9 dropbear
```

Comment:
Can't we remove this line if it's stopped afterwards anyway?

Reply:
I'm not sure, because these `killall` lines are already duplicate in some way, since the function `kill_remaining` afterward is sending `TERM` and KILL to everything anyway...

is it possible that killall handles it better when there are child processes stuck or something?

Reply:
the second condition mostly matches timing(slow) related slow kills... typically for something like collectd...